microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -966,33 +982,37 @@ See also:
 * [Introduction to the Performance tool](../evaluate-performance/index.md)
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 129.
-* This checkbox is present in Microsoft Edge Stable 128.
+* This checkbox is present in Microsoft Edge Canary 131.
+* This checkbox is present in Microsoft Edge Stable 129.
 
 
 <!-- ====================================================================== -->
-## Performance panel: enable sidebar
-<!-- checkbox has no "(?)" link pointing to this anchor wording -->
-
-_new as of Microsoft Edge 128_
+## Performance panel: enable performance insights
 
-Enables the sidebar in the **Performance** tool.
-<!-- todo: details -->
+_new as of Microsoft Edge Stable 129_
 
-<!-- omit See Also -->
+In the **Performance** tool, adds the Performance Insights UI.
+<!-- todo: desc -->
 
-<!-- todo: screenshot of resulting UI, eg in Canary -->
+See also:
+* [Introduction to the Performance tool](../evaluate-performance/index.md)
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 129.
-* This checkbox is present in Microsoft Edge Stable 128.
+* This checkbox is present in Microsoft Edge Canary 131.
+* This checkbox is present in Microsoft Edge Stable 129.
 
 
-<!-- ------------------------------ -->
-#### Enable the experimental, WIP sidebar for the Performance Panel.
-<!-- checkbox has no "(?)" link pointing to this anchor wording -->
-<!-- checkbox label had period -->
+<!-- ====================================================================== -->
+## Performance panel: enable new summary details view for layout shift events
+
+_new as of Microsoft Edge Canary 131_
+
+In the **Performance** tool, adds a summary details view for layout shift events.
```

Comment:
need desc for "Performance panel: enable new summary details view for layout shift events"
@captainbrosset 

https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/experimental-features/?branch=pr-en-us-3283#performance-panel-enable-new-summary-details-view-for-layout-shift-events

Reply:
```suggestion
Shows a **Layout shifts** track in the **Main** section of the **Performance** tool, when viewing a recorded profile. The **Layout shifts** track contains information about when layout shifts occur and which elements are shifted. Unexpected layout shifts can negatively impact the user experience on a webpage.
```